### PR TITLE
feat: Created answers page

### PR DESCRIPTION
**Pull request by arjunbharti**
Created answers page of Quizie

- Total score displayed
- Correct answers displayed

Screenshots: 
Screenshot from 2022-02-19 16-14-27

Preview: https://deploy-preview-5--quiz-with-quizie.netlify.app/pages/answers.html

### CODE REVIEW

**Review by akash-1318 — COMMENTED** [review]
No changes required from my side looks good to merge.

File: pages/answers.html [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+            </div>
+            <div class="question-option">
+                <input type="radio" name="ques-4">
+                <label class="correct-answer">$10,000</label>    
+            </div>
+            <div class="question-option">
+                <input type="radio" name="ques-4">
+                <label>$1,000,000</label>    
+            </div>
+        </div>
```
Comment (HarshnaNHaswani) [inline pages/answers.html:100]: can wrap the answers in a container

File: pages/answers.html [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+                <input type="radio" name="ques-2">
+                <label class="correct-answer">Canada</label>    
+            </div>
+            <div class="question-option">
+                <input type="radio" name="ques-2">
+                <label>Albana</label>    
+            </div>
+        </div>
+        <div class="questions">
+            <p class="question-text">Q. Where did Richard first meet Peter Gregory?</p>
```
Comment (HarshnaNHaswani) [inline pages/answers.html:64]: can use  `<h1>/...<h6>` for questions

**Review by HarshnaNHaswani — APPROVED** [review]
Looks good to me. Just format the file a bit. Nice Work!!

File: pages/answers.html (hunk 1 above)
Comment (arjunbharti) [reply pages/answers.html:100]: noted, thanks

File: pages/answers.html (hunk 2 above)
Comment (arjunbharti) [reply pages/answers.html:64]: noted, thanks

**Pull request merged**